community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/PhysicalLogFileInformation.java

```diff
@@ -30,17 +31,17 @@
 
     private final PhysicalLogFiles logFiles;
     private final LogHeaderCache logHeaderCache;
-    private final TransactionIdStore transactionIdStore;
+    private final Supplier<Long> lastCommittedId;
     private final LogVersionToTimestamp logVersionToTimestamp;
 
     public PhysicalLogFileInformation( PhysicalLogFiles logFiles,
                                        LogHeaderCache logHeaderCache,
-                                       TransactionIdStore transactionIdStore,
+                                       Supplier<Long> lastCommittedId,
```

Comment:
@apcj pointed out that we can copy the pattern used in `LogVersionToTimestamp` rather than passing the supplier. So have introduced `LastEntryInLog` to do that